packages/vuetify/src/directives/ripple/index.ts

```diff
@@ -4,6 +4,8 @@ import './VRipple.sass'
 import { VNode, VNodeDirective } from 'vue'
 import { consoleWarn } from '../../util/console'
 
+const DELAY_RIPPLE = 80
```

Comment:
refer to material-ui
https://github.com/mui-org/material-ui/pull/8605/files#diff-6126a8f8604be55b8d82367154c85852R11